docs/guides/integration-advertiser-dataprovider-overview.md

```diff
@@ -0,0 +1,142 @@
+
+## Advertiser/Data Provider Routes to Use UID2
+
+Within the ad tech industry, advertisers use identity to build audiences, track conversions, and generate their graphs. As an advertiser, or as a data provider acting on behalf of an advertiser, the following table shows some examples of how you can accomplish some of these goals with UID2.
+
+:::note
+There are other ways that you can use UID2, outside these use cases. These are just some examples.
+:::
+
+| Send/Receive? | Action | Advantage/Result |
+| --- | --- | --- |
+| Send in audiences | Send UID2s via API or pixels | Create audiences. |
+| Send in conversions | Send UID2s as conversion information | Use conversion information for measurement (attribution) or for retargeting via API or pixels. |
+| Receive graph data | Receive UID2s from graph/data providers via API or pixels | Build graph data. |
+
+<!-- - **Create/send in audiences**: You can send UID2s to create audiences via API or pixels
+- **Send in conversions**: You can send UID2s as conversion information that can be used for measurement (attribution) or retargeting via API or pixels
+- **Receive graph data**: You can receive UID2s from graph/data providers via API or pixels. -->
+
+## High-Level Steps
+
+At a high level, the steps for advertisers and data providers integrating with UID2 are as follows:
+
+1. Generate raw UID2s from <Link href="../ref-info/glossary-uid#gl-dii">directly identifying information (DII)</Link>, or receive UID2s from another UID2 participant such as a data provider acting on your behalf.
+
+2. Store the raw UID2s and the salt bucket IDs returned from the identity mapping service.
+
+3. Use the UID2s you received in Step 1. For example, you might combine UID2s you generated from DII with other UID2s, or add new UID2s into an existing audience.
+
+4. Use the raw UID2s for some purpose such as:
+
+   - Send stored raw UID2s to DSPs to create audiences and conversions.
+   - Use the raw UID2s for measurement.
+
+5. Monitor for Salt Bucket Rotations for your stored raw UID2s.
+
+6. Periodically, monitor for opt-out status, to be sure that you don't continue using UID2s for users that have recently opted out. For details, see [Monitor for Opt-Out Status](#6-monitor-for-opt-out-status).
+
+## Summary of Implementation Options
+
+The following table shows the implementation options that are available for advertisers and data providers, for each of the high-level steps. Some steps are managed solely as part of your own custom implementation; some steps can be managed by one or more of the UID2 implementation options available. For details, click the link on each step.
+
+| High-Level Step | Implementation Options |
+| --- | --- |
+| [1: Generate Raw UID2s from DII](#1-generate-raw-uid2s-from-dii) | To generate raw UID2s, use any of the following options:<ul><li>Python SDK</li><li>Snowflake</li><li>AWS Entity Resolution</li><li>Raw HTTP endpoint</li></ul> |
```

Comment:
Java SDK works too, maybe linked to the right Java/Py/Snowflake sdks/guides pages directly here?